style/product.css

```diff
@@ -0,0 +1,31 @@
+.filter-sidebar-container{
+    width: 20vw;
+    height: 100%;
+    box-shadow: 5px 5px 10px var(--primary-color-transparent);
+    color: var(--primary-color);
+}
+
+.product-page-container{
+    width: 80vw;
+}
+
+.list-item-container{
+    background-color: transparent;
+    border-radius: 0.5rem;
+    padding-top: 0.5rem;
+    padding-bottom: 0.5rem;
```

Comment:
Use shorthand properties to give value.

Reply:
padding comes from the component library here the x-axis padding is already there I have to change only the top and bottom. If i use shorthand then left and right will become zero